graphql/schema.graphql

```diff
@@ -41,6 +41,12 @@ type Query {
   userConfig: UserConfig
   clientConfig: ClientConfig
   siteBanner: SiteBanner!
+  hostEvents(
```

Comment:
add filter param for `eventId`

Reply:
we ended up deciding on an MVP with no filter